hoot-core-test/src/test/cpp/hoot/core/io/PostgresqlDumpfileWriterTest.cpp

```diff
@@ -122,7 +122,8 @@ class PostgresqlDumpfileWriterTest : public CppUnit::TestFixture
   {
     PostgresqlDumpfileWriter pgDumpWriter;
 
-    QString outFile = "tmp/PostgresqlDumpWriter_out.sql";
+    QDir().mkpath("test-output/io/PostgresqlDumpWriterTest/");
+    QString outFile = "test-output/io/PostgresqlDumpWriterTest/PostgresqlDumpWriter_out.sql";
```

Comment:
Is this a big file? If not, you can use `HOOT_FILE_EQUALS` to compare files rather than manually checking a bunch of lines.